examples/docker/README.md

```diff
@@ -0,0 +1,87 @@
+# Use Packer and cnspec to protect your container image
+
+## Pre-requisites
+
+- [Packer](https://www.packer.io/)
+- [Docker](https://www.docker.com/)
+- Service account with Mondoo Platform
+
+## Run Packer
+
+To run Packer, you need to have a `.pkr.hcl` file. In this example, we have a `docker-ubuntu.pkr.hcl` file.
+
+With the init command, Packer will download the cnspec plugin and install it in the `.packer.d/plugins` directory.
+
+```shell
+packer init docker-ubuntu.pkr.hcl
+```
+
+Configure Mondoo Platform service account credentials either via `cnspec login` or as environment variables.
```

Comment:
nit: Should we expect people to know how this works at this point or should we link to our docs?

Reply:
We should ensure it is in the docs and reference that.